### PR TITLE
Create Book class

### PR DESCRIPTION
**Pull request by grabowskija**
Added class Book with getters and setters

### CODE REVIEW

File: JavaDabaDoo_console/src/main/java/com/infoshareacademy/Book.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import java.util.List;
+
+public class Book {
+    private long id;
+    private String isbn;
+    private String title;
+    private String author;
+    private String language;
+    private String category;
+    private List<Integer> scores = new ArrayList<>();
```
Comment (grabowskija) [inline JavaDabaDoo_console/src/main/java/com/infoshareacademy/Book.java]: i have own doubt how it behave  while serialization of the book objects

File: JavaDabaDoo_console/src/main/java/com/infoshareacademy/Book.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    private LocalDateTime dateOfAdd = LocalDateTime.now();
```
Comment (grabowskija) [inline JavaDabaDoo_console/src/main/java/com/infoshareacademy/Book.java]: same as perv comment

File: JavaDabaDoo_console/src/main/java/com/infoshareacademy/Book.java (hunk 1 above)
Comment (michal-mrozek) [reply JavaDabaDoo_console/src/main/java/com/infoshareacademy/Book.java]: I used maps within the objects and serialization worked fine with it.
We will find out.

**Review by michal-mrozek — COMMENTED** [review]
1. I am thinking about localDateTime class, Maybe we would like to use a simple string to present the time (actually the date) f.e dd/mm/yyyy? 
2. We will be needed constructor or two.

**Comment by grabowskija** [comment]
1.We can do the math on localDateTime on string probably not so i think some representation o time is better.
2. But with what paremeters ? all of them ?;) To serialize/deserialize i think the default one is needed.

**Comment by michal-mrozek** [comment]
1. Fair point.
2. 1st default, 2nd with all parameters? At least, we will have choice how to create the object.

**Comment by grabowskija** [comment]
I have added long id, String isbn, String title, String author, String language, String category. The scores and dateOfAdd should be initialized without it i think.

**Comment by michal-mrozek** [comment]
Score and date still should be initialled within constructor but with the fix value as empty arraylist and time of creating the object date now().

**Comment by grabowskija** [comment]
fixed :)

**Comment by marcin-korzeniewski** [comment]
+1

**Pull request merged**